src/View/Components/Step/Items.php

```diff
@@ -8,7 +8,7 @@
 class Items extends BaseComponent
 {
     public function __construct(
-        public ?string $step = null,
+        public int $step,
```

Comment:
I think we need to declare this as an optional parameter. What will happen if it is not declared?

Reply:
In fact, this cannot be optional, because to be able to navigate correctly you must pass the integer, as this is used for increment and decrement.